File: scripts/algorithms/F/Final Value of Variable After Performing Operations/Final Value of Variable After Performing Operations.cpp
// Runtime: 8 ms (Top 48.49%) | Memory: 14.60 MB (Top 7.99%)

class Solution {
public:
    int finalValueAfterOperations(vector<string>& o,int c=0) {
           for(auto &i:o) if(i=="++X" or i=="X++") c++; else c--;
           return c;
    }
};

File: scripts/algorithms/F/Final Value of Variable After Performing Operations/Final Value of Variable After Performing Operations.py
# Runtime: 111 ms (Top 13.83%) | Memory: 14 MB (Top 12.56%)
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        x = 0
        for o in operations:
            if '+' in o:
                x += 1
            else:
                x -= 1
        return x